test/unittests/evm_test.cpp

```diff
@@ -1171,16 +1173,24 @@ TEST_F(evm, memory_access)
 
             code += bytecode{t.opcode};
 
-            execute(code);
+            auto const gas = 30000000;
```

Comment:
Can you try `8796294610952`. That's the max value causing all test to go OOG

Reply:
evmone passes, interpreter returns `EVMC_INVALID_MEMORY_ACCESS`

Reply:
Is this solved now?

Reply:
No, interpreter checks the size of returndata first and returns `EVMC_INVALID_MEMORY_ACCESS` before checking for required memory size

Reply:
Can we apply something like this then? https://github.com/ethereum/evmone/pull/92

It also looks `RETURNDATACOPY` might require its own test for `OUT_OF_GAS` not being `INVALID_MEMORY_ACCESS`.

Reply:
Yes that, helps.

I rebased this PR on #92

Reply:
Ready now?

Reply:
Yes